tools/dumppdf.py

```diff
@@ -148,7 +150,7 @@ def resolve_dest(dest):
                         if subtype and repr(subtype) == '/GoTo' and action.get('D'):
                             dest = resolve_dest(action['D'])
                             pageno = pages[dest[0].objid]
-                outfp.write('<outline level="%r" title="%s">\n' % (level, q(s)))
+                outfp.write('<outline level="%r" title="%s">\n' % (level, title))
                 if dest is not None:
                     outfp.write('<dest>')
                     dumpxml(outfp, dest)
```

Comment:
Since checks were added relatively late, we ran these manually through flake8. This was the output for this particular file:
```
.\tools\dumppdf.py:18:1: F401 'pdfminer.utils.q' imported but unused
.\tools\dumppdf.py:22:1: E302 expected 2 blank lines, found 0
.\tools\dumppdf.py:40:15: E231 missing whitespace after ','
.\tools\dumppdf.py:71:80: E501 line too long (84 > 79 characters)
.\tools\dumppdf.py:94:1: E302 expected 2 blank lines, found 1
.\tools\dumppdf.py:102:1: E302 expected 2 blank lines, found 1
.\tools\dumppdf.py:107:32: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:111:31: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:122:1: E302 expected 2 blank lines, found 1
.\tools\dumppdf.py:127:22: E201 whitespace after '('
.\tools\dumppdf.py:127:56: E231 missing whitespace after ','
.\tools\dumppdf.py:128:62: E202 whitespace before ')'
.\tools\dumppdf.py:129:9: E306 expected 1 blank line before a nested definition, found 0
.\tools\dumppdf.py:141:23: E231 missing whitespace after ','
.\tools\dumppdf.py:141:29: E231 missing whitespace after ','
.\tools\dumppdf.py:141:34: E231 missing whitespace after ','
.\tools\dumppdf.py:141:36: E231 missing whitespace after ','
.\tools\dumppdf.py:150:80: E501 line too long (84 > 79 characters)
.\tools\dumppdf.py:153:80: E501 line too long (81 > 79 characters)
.\tools\dumppdf.py:168:1: E305 expected 2 blank lines after class or function definition, found 1
.\tools\dumppdf.py:170:1: E302 expected 2 blank lines, found 0
.\tools\dumppdf.py:182:80: E501 line too long (82 > 79 characters)
.\tools\dumppdf.py:198:80: E501 line too long (81 > 79 characters)
.\tools\dumppdf.py:203:1: E302 expected 2 blank lines, found 1
.\tools\dumppdf.py:213:24: E231 missing whitespace after ','
.\tools\dumppdf.py:225:30: E231 missing whitespace after ','
.\tools\dumppdf.py:233:5: E306 expected 1 blank line before a nested definition, found 0
.\tools\dumppdf.py:234:80: E501 line too long (88 > 79 characters)
.\tools\dumppdf.py:235:16: E127 continuation line over-indented for visual indent
.\tools\dumppdf.py:241:16: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:252:21: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:253:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:254:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:255:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:255:40: E201 whitespace after '('
.\tools\dumppdf.py:255:71: E202 whitespace before ')'
.\tools\dumppdf.py:256:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:256:39: E201 whitespace after '('
.\tools\dumppdf.py:256:68: E202 whitespace before ')'
.\tools\dumppdf.py:257:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:258:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:259:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:260:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:261:23: E701 multiple statements on one line (colon)
.\tools\dumppdf.py:274:1: E305 expected 2 blank lines after class or function definition, found 1
.\tools\dumppdf.py:274:26: E701 multiple statements on one line (colon)
```
you should address these before we approve.